test/cql-pytest/custom_decorators.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from test_config import IS_SCYLLA_VERSION
+
+
+# The "scylla_only" decorator can be used by tests for Scylla-only features,
+# which do not exist on Apache Cassandra. A test using this decorator will be
+# skipped if running with "run-cassandra".
+scylla_only = pytest.mark.skipif(condition=not IS_SCYLLA_VERSION,
+                                 reason="The test can run only with Scylla version")
```

Comment:
The phrase "Scylla version" sounds strange. It sounds like it is missing a version number (e.g., "Scylla version 3"). 
The message we already had in `scylla_only` was "Scylla-only test skipped". Isn't that clearer and shorter?
I would also rename the is_scylla_version name. Why isn't is_scylla enough?